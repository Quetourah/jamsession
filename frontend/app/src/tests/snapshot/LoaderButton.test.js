import React from 'react';
import LoaderButton from '../../components/LoaderButton';
import renderer from 'react-test-renderer';

it('renders loader button correctly', () => {
  const tree = renderer
    .create(<LoaderButton/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});