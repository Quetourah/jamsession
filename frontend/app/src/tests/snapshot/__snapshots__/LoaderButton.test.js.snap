// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loader button correctly 1`] = `
<button
  className="LoaderButton  btn btn-default"
  disabled={false}
  type="button"
/>
`;
